src/express-tooling.ts

```diff
@@ -28,82 +29,102 @@ import { safeLogger } from './safe-logger';
 const AUTHORIZATION_HEADER_FIELD_NAME = 'authorization';
 
 /**
- * Returns a function (express middleware) that validates the scopes against the user scopes
- * attached to the request (for example by `handleOAuthRequestMiddleware`).
- * If the the requested scopes are not matched request is rejected (with 403 Forbidden).
+ * Returns a function (express middleware) that validates the scopes against the user scopes.
```

Comment:
"[...]validates the scopes against the user scopes"
Just some wording: what exactly are "the scopes" and "the user scopes" and where do they come from?
Suggestion:
"[...]validates the scopes attached to the request against the specified required scopes"